bin/load.js

```diff
@@ -53,6 +54,7 @@ const FILE_MODULES = {
     ncit,
     ncitFdaXref,
     ontology,
+    sources,
```

Comment:
Not a blocker, but since there is no file to load (right now, all_sources/index.js is directly importing data/sources.js), it forces the user still add an actual filepath (e.g. node bin/load.js -g http://$HOSTNAME:8080/api -u $USER -p $JIRA_PASS file sources src/sources.js)  in order to run the loader. Could be its own type of loader, like the civic one.